configure.ac

```diff
@@ -64,6 +77,17 @@ if [[ -f "$header_prefix/include/libzfs/libzfs.h" ]] || [[ "$zof" = true ]]; the
 "
 	LDFLAGS="-L${header_prefix}/lib ${LDFLAGS}"
 
+elif [[ "$build_macos" = "yes" ]] && [[ "x$with_o3x" != "x"  ]]; then
+	zof=true
+
+	CFLAGS="${CFLAGS} 
+-I${with_o3x}/include/
+-I${with_o3x}/lib/libspl/include/
+-I./include/
+-D_MACHINE_ENDIAN_H_ -DHAVE_ISSETUGID
+-include ${with_o3x}/include/sys/zfs_mount.h
+"
+	LDFLAGS="-L${header_prefix}/lib ${LDFLAGS}"
```

Comment:
`header_prefix` is being used here instead of `with_o3x`, is this intentional ?

Reply:
Contrary to the headers, libraries are installed in the standard location.